src/main.rs

```diff
@@ -179,6 +187,10 @@ fn main() -> ! {
                     led_color = (r, g, b);
                     led_pulse = pulse;
                 },
+                Command::Bootload => {
+                    led.set_high().unwrap();
+                    bootload::request_bootloader();
```

Comment:
`return bootload::request_bootloader();` to make it clear that this is the end of the loop?

Reply:
`request_bootloader` returns the [never type](https://doc.rust-lang.org/reference/types/never.html) which means it will never return control flow to this call, which means the `return` code itself will not run. I actually get a warning about unreachable code if I add a `return` here.